src/hevm/test/test.hs

```diff
@@ -718,8 +713,8 @@ tests = testGroup "hevm"
           }
         }
         |]
-      [Qed res] <- withSolvers Z3 1 Nothing $ \s -> checkAssert s defaultPanicCodes c (Just ("checkval(uint256,uint256)", [AbiUIntType 256, AbiUIntType 256])) [] VeriOpts {simp = False, debug = False, maxIter = Nothing, askSmtIters = Nothing}
-      putStrLn $ "successfully explored: " <> show (Expr.numBranches res) <> " paths"
+      (_, [Qed _]) <- withSolvers Z3 1 Nothing $ \s -> checkAssert s defaultPanicCodes c (Just ("checkval(uint256,uint256)", [AbiUIntType 256, AbiUIntType 256])) [] VeriOpts {simp = False, debug = False, maxIter = Nothing, askSmtIters = Nothing}
+      putStrLn "this should always be true, due to bitwise OR with positive value"
```

Comment:
This printing seems misleading. That text I think is from some other test? This test does not seem to use bitwise OR I think?